subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -174,6 +175,11 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
 
         then:
         scanRunner()
+            .expectLegacyDeprecationWarningIf(versionNumber < FIRST_VERSION_CALLING_BUILD_PATH,
+                "Gradle Enterprise plugin $version has been deprecated. " +
+                    "Starting with Gradle 9.0, only Gradle Enterprise plugin 3.13.1 or newer is supported. " +
+                    "Consult the upgrading guide for further information: https://docs.gradle.org/${GradleVersion.current().version}/userguide/upgrading_version_8.html#unsupported_ge_plugin_3.13"
```

Comment:
```suggestion
                "Using Gradle Enterprise plugin $version has been deprecated. " +
                    "Starting with Gradle 9.0, only Gradle Enterprise plugin 3.13.1 or newer will be supported. " +
                    "Consult the upgrading guide for further information: https://docs.gradle.org/${GradleVersion.current().version}/userguide/upgrading_version_8.html#unsupported_ge_plugin_3.13"
```

💄 A suggestion for a little bit different wording, use at your own discretion.

Reply:
@wolfs, @alllex's version is more future-proof as far as wording goes.